PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,11 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            double percent = (double)numberCorrectAnswers / (double)numberOfQuestions;
+            percent = percent * 100;
+            string percentage = percent + "%";
+            //return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return percentage;        
```

Comment:
Not a bad solution here. I would have simplified it and also added a Math.Round in case the result still wasn't a whole number
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```